packages/editor/src/components/post-preview-button/index.js

```diff
@@ -210,17 +211,23 @@ export default compose( [
 			getEditedPostPreviewLink,
 		} = select( 'core/editor' );
 		const {
+			getAutosave,
 			getPostType,
+			hasFetchedAutosave,
 		} = select( 'core' );
 
 		const previewLink = getEditedPostPreviewLink();
 		const postType = getPostType( getEditedPostAttribute( 'type' ) );
+		const post = getCurrentPost();
+		const autosave = getAutosave( post );
+
 		return {
+			autosave,
```

Comment:
Why is this being passed as a prop now? Especially considering the component already receives `autosave` from `withDispatch` as a _function_, which this is not.

Reply:
Yeah, that's a mistake, I've removed it.